src/components/taskDetails/Details.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import setColor from './DetailsColor';
+import classNames from './task-details.module.scss';
+
+type Props = {
+  detailType: string;
+  value: string;
+};
+
+const Details: FC<Props> = ({ detailType, value }) => {
+  return (
+    <div>
+      <span className={classNames['detail_type']}>{detailType}:</span>
+      <span
+        className={classNames['detail_value']}
+        style={{ color: setColor[value] ? setColor[value] : 'black' }}
```

Comment:
Nit: Can we apply [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) here?
```suggestion
        style={{ color: setColor[value] ?? 'black' }}
```

Reply:
done. Thanks for the suggestion. Learnt something new today.